src/pluginManager.ts

```diff
@@ -80,7 +80,7 @@ export class PluginManager {
   }
 
   public static isQualifiedPluginIdentifier(identifier: string): boolean {
-    return PluginManager.PLUGIN_IDENTIFIER_PATTERN.test(identifier);
+    return PluginManager.PLUGIN_IDENTIFIER_PATTERN.test(identifier.replace(path.sep, "/"));
```

Comment:
Windows will use `\` in paths while the npm package names will always use `/` as the scope seperator. Package names cannot contain `\` so this won't conflict with anything else.

Reply:
I don't like the fact that we put that in to the isQualifiedPluginIdentifier function. And the backslash part is only really used in the context of plugin loading. Maybe we could add the replace just at the places this function is called in context of dealing with paths?